core/idl/wallet/ethereum/ethereum_like_wallet.djinni

```diff
@@ -151,6 +169,11 @@ EthereumLikeAccount = interface +c {
     # This estimation is based on X last incoming txs (to address) that succeeded
     # Note: same note as above
     getEstimatedGasLimit(address: string, callback: Callback<BigInt>);
+    # Get estimated gas limit to set so the transaction will succeed
+    # The passed address could be EOA or contract
+    # This estimation is based on a dry-run on the node, and it will fail if the request is ill-formed
+    # Note: same note as above
+    getDryrunGasLimit(address: string, request: EthereumGasLimitRequest, callback: Callback<BigInt>);
```

Comment:
Please change `getDryrunGasLimit` to `getDryRunGasLimit` to match our current naming convention